PixiEditor/PixiEditor.csproj

```diff
@@ -14,6 +14,7 @@
     <PackageIcon>icon.ico</PackageIcon>
     <ApplicationIcon>..\icon.ico</ApplicationIcon>
     <Authors>Krzysztof Krysiński</Authors>
+    <StartupObject />
```

Comment:
What's the reason of adding `<StartupObject />` here?

Reply:
Hey, I'm not sure how that appeared there since I never messed with StartupObject in the properties but considering it's empty my best guess is that it was auto generated by visual studio. I went ahead and removed that though, and fixed the output type.